src/data/maestro.py

```diff
@@ -0,0 +1,16 @@
+import os
+import tensorflow as tf
+import data.process as pro
+
+def maestro_from_files(root_path, frame_size):
+    dataset = tf.data.Dataset.list_files(os.path.join(root_path, '**/*.wav'))
+    dataset = pro.pipeline(dataset, [
```

Comment:
I though we refactored that to `pipeline([...])(dataset)`?

Reply:
I noticed this bug in my branch as well but fixed it.